.circleci/config.yml

```diff
@@ -799,6 +804,13 @@ jobs:
               echo Last build occured today, halting.
               circleci step halt
             fi
+            echo  echo Last build occured $LAST, building.
+            date +%Y-%M-%d > /home/circleci/lastbuild
+      - save_cache:
+          name: store updated daily latch file.
+          key: code-coverage-daily-{{ epoch }}
+          paths:
+            - /home/circleci/lastbuild
```

Comment:
I think having the update in a separate step was better, but I don't feel super strongly about it.